Troubleshooting.md

```diff
@@ -0,0 +1,84 @@
+# Troubleshooting
+
+This document will help troubleshoot the common debugging errors. If you still cannot solve your problems, please [submit an issue](https://github.com/Microsoft/vscode-java-debug/issues).
+
+## Java Language Support extension fails to start.
+Java Debugger is leveraging `"Language Support for Java(TM) by Red Hat"` extension for source mapping and project support. If Java Language Support extension fails to start, Java Debugger may not work as expected. Here is a simple way to check the Language Server is started. Open a Java or pom.xml file, and then check the status icon at the right corner of VS Code status bar is thumbs up or not.
+
+ ![ status indicator ](https://raw.githubusercontent.com/redhat-developer/vscode-java/master/images/statusMarker.png).
+
+### Try:
+1. If you get the error `"The JAVA_HOME environment variable points to a missing folder"` or `"Java runtime could not be located"`, please make sure the JAVA_HOME and bin path point to a valid JDK home directory. Otherwise, ignore this step.
+2. Open VS Code Command Palette (`Ctrl+Shift+P`), and run command `"Java: Update project configuration"` to force the Language Support extension to resolve the project relationship again.
+3. If the step above doesn't work, please try to clean the stale workspace cache. Close all opened VS Code windows first and delete the following cache directory:

```

Comment:
If the step above doesn't work, please try to clean the stale workspace cache.
->
Try cleaning the stale workspace cache.

Reply:
Quit all VS Code processes and clean the following cache directories.